raster/src/test/scala/geotrellis/raster/io/geotiff/BitGeoTiffTileSpec.scala

```diff
@@ -82,5 +76,22 @@ class BitGeoTiffTileSpec extends FunSpec
 
       assertEqual(res, tile)
     }
+
+    it("should convert tiled tiffs") {
+      val tiff = SinglebandGeoTiff(geoTiffPath("bilevel.tif"))
+      val tile = tiff.tile.toArrayTile() // .convert(BitCellType)
+
+      // check that it is possible to convert bit cellType to bit cellType
+      val tiffTile = tile.toGeoTiffTile(GeoTiffOptions.DEFAULT).convert(BitCellType)
+      assertEqual(tiffTile.toArrayTile(), tile.toArrayTile)
+
+      // check that it is possible to convert int cellType to bit cellType
+      // and that bitCellType conversion is idempotent
```

Comment:
This requires `idempotence` check, since it is the best way to prove that we're setting / unsetting and inverting bits correct to check that even / odd number of conversions would not affect the final result.